### ISSUE TITLE
Tracking for auto-reported issue 1066859

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#2391]
In case it needs more details, please let me know.

https://ea.jetbrains.com/browser/ea_reports/1066859

**Comment by zolotov** [comment]
@ignatov have you seen this before? It just shouldn't happen

**Comment by dlsniper** [comment]
No, it's the first time. I've opened the IDE, I've got the update to the 2016.1 RC 2 version dialog, there was another dialog to update another plugin and, after the IDE started indexing I've received this shortly. 

I've looked in the log files and I get a lot of errors about not saving settings: http://pastebin.com/u3Nt5bJe

Maybe it's because I'm testing on IDEA 2016.1?

**Comment by dlsniper** [comment]
I've restarted the IDE and the save settings issue is not present anymore and I've yet to see the original issue so far.
